telecine/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+  <string name="app_name" translatable="false">Telecine</string>
+  <string name="launcher_name" translatable="false">@string/app_name</string>
```

Comment:
Delete these two strings. They are not meant to be translated and should only be defined in the `values/strings.xml`.